src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -23,8 +25,8 @@
     }
 
     @RequestMapping(method = RequestMethod.GET, value = "getTask")
-    public TaskDto getTask() {
-        return new TaskDto(1L, "test title", "test_content");
+    public TaskDto getTask(@RequestParam String id) {
+        return taskMapper.mapToTaskDto(service.getTask(Long.parseLong(id)));
```

Comment:
jest okej, jak w kursie, aczkowliek w kontrolerach nie chcemy robić żadnych mapowań. To zadanie serwisu, tutaj `DbService`